packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -399,39 +405,40 @@ describe("IModelTransformer", () => {
         "==============================="
       );
       const targetImporter = new RecordingIModelImporter(targetDb);
-      const transformer = new TestIModelTransformer(sourceDb, targetImporter);
+      const transformer = new TestIModelTransformer(sourceDb, targetImporter, {
+        forceExternalSourceAspectProvenance: true,
+      });
       await transformer.processAll();
       assert.equal(targetImporter.numModelsInserted, 0);
       assert.equal(targetImporter.numModelsUpdated, 0);
       assert.equal(targetImporter.numElementsInserted, 1);
       assert.equal(targetImporter.numElementsUpdated, 33);
-      // FIXME: upgrade this test to use a briefcase so that we can detect element deletes
-      // use the new force old detect deletes behavior flag here
-      // assert.equal(targetImporter.numElementsDeleted, 5);
+      /**
+       * There are 5 elements deleted in TransformerExtensiveTestScenario.updateDb, but only 4 detected.
+       * This is because PhysicalObject6's code is scoped to PhysicalObject5. When PhysicalObject5 is deleted, PhysicalObject6 is also deleted in the superclasses
+       * of the RecordingIModelImporter and therefore can't be detected by the RecordingIModelImporter.
+       */
+      assert.equal(targetImporter.numElementsDeleted, 4);
       assert.equal(targetImporter.numElementAspectsInserted, 0);
       assert.equal(targetImporter.numElementAspectsUpdated, 2);
       assert.equal(targetImporter.numRelationshipsInserted, 2);
       assert.equal(targetImporter.numRelationshipsUpdated, 1);
-      // FIXME: upgrade this test to use a briefcase so that we can detect element deletes
-      // assert.equal(targetImporter.numRelationshipsDeleted, 0);
+
+      assert.equal(targetImporter.numRelationshipsDeleted, 1);
       targetDb.saveChanges();
-      // FIXME: upgrade this test to use a briefcase so that we can detect element deletes
-      TransformerExtensiveTestScenario.assertUpdatesInDb(
-        targetDb,
-        /* FIXME: */ false
-      ); // Switch back to true once we have the old detect deltes behavior flag here. also enable force old provenance method.
-      // which is only used in in-imodel transformations.
+      TransformerExtensiveTestScenario.assertUpdatesInDb(targetDb, true);
 
       assert.equal(
         numTargetRelationships +
           targetImporter.numRelationshipsInserted -
           targetImporter.numRelationshipsDeleted,
         count(targetDb, ElementRefersToElements.classFullName)
       );
-      // FIXME: why?
+      // FIXME<NICK>: I saw that the previous commit which added this FIXME: why changed the test from 3 to 2. But now in my test its producing 2.
```

Comment:
Calling attn to this comment

Reply:
`TargetInformationRecord` elements seem to come from `TestIModelTransformer.initClassRemapping` which remaps all `SourceInformationRecord`s to it. Other tests find 3 of them so I'd be concerned about the 2 as well.
We should check how many `SourceInformationRecords` there are in the source (I assume 3) and see if we deleted any of them.

Reply:
I checked and we do delete one, updated the comment